src/Entity/WmContentContainer.php

```diff
@@ -139,6 +153,20 @@ public function getHostBundles()
     }
 
     /**
+     * @return array
```

Comment:
@brownrl This function seems identical to getChildBundlesAll except for the bundle where you're getting bundles for. Maybe make one function with param?